google/cloud/aiplatform/metadata/artifact.py

```diff
@@ -262,37 +262,44 @@ def create(
         project: Optional[str] = None,
         location: Optional[str] = None,
         credentials: Optional[auth_credentials.Credentials] = None,
+        base_artifact_schema: Optional[base_artifact.BaseArtifactSchema] = None,
     ) -> "Artifact":
         """Creates a new Metadata Artifact.
 
         Args:
             schema_title (str):
-                Required. schema_title identifies the schema title used by the Artifact.
-
+                Optional. schema_title identifies the schema title used by the
```

Comment:
My UX preference is to separate this into a separate create method to avoid overloading this method. ie:

```
def create(cls, schema_title, ...)

def create_from_base_schema(cls, base_artifact_schema=...)
```

@jialuzh

Reply:
Done, I actually like this since approach more since it detaches the execution flow from the main execution and lowering the risk of this SDK change. Also this change is invisible to the users since BaseArtifact.create() calls this method internally.